fahrschein-jdbc/src/main/java/org/zalando/fahrschein/jdbc/TransactionalBatchHandler.java

```diff
@@ -0,0 +1,16 @@
+package org.zalando.fahrschein.jdbc;
```

Comment:
I would consider this class more as an example, you might want to customize the rollback behaviour.